docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -16,16 +13,133 @@ This document covers a range of common scenarios using the PXIe-148X Generation
 ---
 
 ## Prerequisites
+This tutorial is written for users who understand how to perform a basic generation and a basic acquisition with PXIe-148X GMSL or FPD-Link interface modules. It is recommended to complete the [PXIe-148X Getting Started Example - Basic Acquisition Tutorial](gse-acq-basic.md) and [PXIe-148X Getting Started Example - Basic Generation Tutorial](gse-gen-basic.md) before attempting this tutorial.
+
+> Note: The tutorials in this document assume the use of a PXIe-148X GMSL or FPD-Link Generation Interface module and an Acquisition Interface module connected to SI0 and SO0 of the respective modules. A Leopard Imaging IMX490 camera is also used to acquire packet data and replay it through a generation interface module. See PXIe-148X Getting Started Example - Basic Acquisition Tutorial and Basic Generation Tutorial for specific setup if needed.
 
 ## Generation and Acquisition Topics
+This tutorial expects that SO0 of a generation card and SI0 of an acquisition card are cabled together. This tutorial will use one generation LabVIEW project and one acquisition LabVIEW project to demonstrate the generation capabilities of the 148x generation interface devices.
+
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+
 
 ### Performing a Simple Generation and Acquisition
+1. Double click the Create CSI-2 Packet TDMS Files VI in the generation LabVIEW project and create a TDMS file for SI0. Leave all values default.
+
+> Note: Refer to the Basic Generation Tutorial to create a TDMS file for Serial Output channel 0.
```

Comment:
Can linkify 'Basic Generation Tutorial' and rename to 'PXIe-148X Getting Started Example - Basic Generation Tutorial' for consistency in other locations in the doc